### PR TITLE
Fixes coming from the openDS public review

### PR DESCRIPTION
**Pull request by samleeflang**
Will fix the following tickets:
- https://github.com/DiSSCo/openDS/issues/161
- https://github.com/DiSSCo/openDS/issues/148
- https://github.com/DiSSCo/openDS/issues/149
- https://github.com/DiSSCo/openDS/issues/147
- https://github.com/DiSSCo/openDS/issues/190
- https://github.com/DiSSCo/openDS/issues/183
- https://github.com/DiSSCo/openDS/issues/163

Trying to keep the amount of PR's low, not to unnessecary burden.
But do like to keep an administration on which tickets I got

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -476,8 +476,9 @@ def generate_terms(header_mdfile, term_file):
         on=['compound_name'], how='left'
     )
     terms = terms_skos_df.sort_values(by=['class_name'])
-    terms['examples'] = terms['examples'].str.replace(r'"', '')
-    terms['definition'] = terms['definition'].str.replace(r'"', '')
+    terms['examples'] = terms['examples'].str.replace(r'"', '').replace("'", "").replace("[", "").replace("]", "")
+    terms['enum'] = terms['enum'].str.replace(r'"', '').replace("'", "").replace("[", "").replace("]", "")
+    terms['definition'] = terms['definition'].str.replace(r'"', '').replace("'", "").replace("[", "").replace("]", "")
```
Comment (TomDijkema) [inline app/routes.py:481]: Maybe make a function out of this as you are repeating the same action three times

File: app/templates/includes/termlist/termlist-table.html [hunk 2]
```diff
@@ -64,6 +64,22 @@ <h2 id="5-vocabulary">5 Vocabulary</h2>
...
+							<th scope="row">Controlled Vocabulary</th>
+							<td>
+								<ul class="td-list">
+									{% for item in list1 %}
+										<li><span>{{ item }}</span></li>
+									{% endfor %}
+								</ul>
+
+							</td>
+						</tr>
```
Comment (TomDijkema) [inline app/templates/includes/termlist/termlist-table.html:80]: Don't know how this should look but often <th> and <td> elements are stored in separate <tr> (table rows)

**Review by southeo — COMMENTED** [review]
:pie:

File: app/routes.py (hunk 1 above)
Comment (samleeflang) [reply app/routes.py:481]: :+1:

File: app/templates/includes/termlist/termlist-table.html (hunk 2 above)
Comment (samleeflang) [reply app/templates/includes/termlist/termlist-table.html:80]: Hmm took this over from Latimer Core only made some small changes to it. Would like to keep this for now

File: app/data/opends/create-update-tombstone-event-termlist.csv [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+ods:,roleOrder,Role Order,Order of the agent in the role. Can be used to indicate the order of importance,,,"1, 2",http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,Agent,False,False,Agent.roleOrder,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/roleOrder,ods:roleOrder,integer,
+schema:,email,Email,"Email of the agent, can be present in case the agent is a maintainer of a MAS",,,,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,Agent,False,False,Agent.email,http://schema.org/,http://schema.org/email,schema:email,string,
+schema:,url,URL,"URL of the agent, can be present in case the agent is a maintainer of a MAS",,,,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,Agent,False,False,Agent.url,http://schema.org/,http://schema.org/url,schema:url,string,
+ods:,hasIdentifier,Has Identifier,Contains zero or more ods:Identifier objects,,,,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,Agent,False,False,Agent.hasIdentifier,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/hasIdentifier,ods:hasIdentifier,array<ods:Identifier>,
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply app/data/opends/create-update-tombstone-event-termlist.csv:24]: Why would we need it in Annotation or in the CUT Event. Is it possible for them to have multiple identifiers. It currently makes the assumption that there is only one unique identifier (handle) for the Annotation, CUT, DataMapping, SourceSystem. Specimen and Media have multiple identifiers (making our lives harder) and therefore the `@id` and `dcterms:identifier` properties aren't enough to capture all identifiers

File: app/data/opends/annotation-termlist.csv [hunk 4]
```diff
@@ -1,47 +1,47 @@
...
+ods:,tombstoneText,Tombstone Text,A reason why the Digital Object was tombstoned,,,"This is a duplicate of XXX, This object was made by mistake",http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,TombstoneMetadata,True,False,TombstoneMetadata.tombstoneText,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/tombstoneText,ods:tombstoneText,string,
+ods:,TombstonedByAgent,Tombstoned By Agent,"The agent who tombstoned the object, contains an ods:Agent object",,,,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,TombstoneMetadata,True,False,TombstoneMetadata.TombstonedByAgent,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/TombstonedByAgent,ods:TombstonedByAgent,object,
+ods:,hasRelatedPID,Has Related Pid,The PIDs of the object the tombstoned object is related to,,,,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,TombstoneMetadata,False,False,TombstoneMetadata.hasRelatedPID,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/hasRelatedPID,ods:hasRelatedPID,array<object>,
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply app/data/opends/annotation-termlist.csv:30]: Fixed

File: app/data/opends/data-mapping-termlist.csv [hunk 5]
```diff
@@ -1,29 +1,29 @@
...
+ods:,tombstoneText,Tombstone Text,A reason why the Digital Object was tombstoned,,,"This is a duplicate of XXX, This object was made by mistake",http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,TombstoneMetadata,True,False,TombstoneMetadata.tombstoneText,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/tombstoneText,ods:tombstoneText,string,
+ods:,TombstonedByAgent,Tombstoned By Agent,"The agent who tombstoned the object, contains an ods:Agent object",,,,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,TombstoneMetadata,True,False,TombstoneMetadata.TombstonedByAgent,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/TombstonedByAgent,ods:TombstonedByAgent,object,
+ods:,hasRelatedPID,Has Related Pid,The PIDs of the object the tombstoned object is related to,,,,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,TombstoneMetadata,False,False,TombstoneMetadata.hasRelatedPID,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/hasRelatedPID,ods:hasRelatedPID,array<object>,
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply app/data/opends/data-mapping-termlist.csv:20]: Fixed

File: app/data/opends/digital-media-termlist.csv [hunk 6]
```diff
@@ -1,89 +1,89 @@
...
+ods:,tombstoneText,Tombstone Text,A reason why the Digital Object was tombstoned,,,"This is a duplicate of XXX, This object was made by mistake",http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,TombstoneMetadata,True,False,TombstoneMetadata.tombstoneText,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/tombstoneText,ods:tombstoneText,string,
+ods:,TombstonedByAgent,Tombstoned By Agent,"The agent who tombstoned the object, contains an ods:Agent object",,,,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,TombstoneMetadata,True,False,TombstoneMetadata.TombstonedByAgent,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/TombstonedByAgent,ods:TombstonedByAgent,object,
+ods:,hasRelatedPID,Has Related Pid,The PIDs of the object the tombstoned object is related to,,,,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,TombstoneMetadata,False,False,TombstoneMetadata.hasRelatedPID,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/hasRelatedPID,ods:hasRelatedPID,array<object>,
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply app/data/opends/digital-media-termlist.csv:89]: Fixed

File: app/data/opends/digital-specimen-termlist.csv [hunk 7]
```diff
@@ -1,311 +1,319 @@
...
+ods:,tombstoneText,Tombstone Text,A reason why the Digital Object was tombstoned,,,"This is a duplicate of XXX, This object was made by mistake",http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,TombstoneMetadata,True,False,TombstoneMetadata.tombstoneText,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/tombstoneText,ods:tombstoneText,string,
+ods:,TombstonedByAgent,Tombstoned By Agent,"The agent who tombstoned the object, contains an ods:Agent object",,,,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,TombstoneMetadata,True,False,TombstoneMetadata.TombstonedByAgent,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/TombstonedByAgent,ods:TombstonedByAgent,object,
+ods:,hasRelatedPID,Has Related Pid,The PIDs of the object the tombstoned object is related to,,,,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,TombstoneMetadata,False,False,TombstoneMetadata.hasRelatedPID,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/hasRelatedPID,ods:hasRelatedPID,array<object>,
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply app/data/opends/digital-specimen-termlist.csv:319]: Fixed

File: app/routes.py [hunk 8]
```diff
@@ -476,8 +476,9 @@ def generate_terms(header_mdfile, term_file):
         on=['compound_name'], how='left'
     )
     terms = terms_skos_df.sort_values(by=['class_name'])
-    terms['examples'] = terms['examples'].str.replace(r'"', '')
-    terms['definition'] = terms['definition'].str.replace(r'"', '')
+    terms['examples'] = terms['examples'].str.replace(r'"', '').replace("'", "").replace("[", "").replace("]", "")
+    terms['enum'] = terms['enum'].str.replace(r'"', '').replace("'", "").replace("[", "").replace("]", "")
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply app/routes.py:480]: :+1:

File: app/data/opends/create-update-tombstone-event-termlist.csv [hunk 9]
```diff
@@ -1,24 +1,24 @@
...
+rdfs:,comment,Comment,"A comment about the activity, https://www.w3.org/2000/01/rdf-schema#comment",,,This activity was created by the user,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,Activity,False,False,prov:Activity.comment,http://www.w3.org/2000/01/rdf-schema#,http://www.w3.org/2000/01/rdf-schema#comment,rdfs:comment,string,
+ods:,changeValue,Change Value,"The json patch object that describes the changes made to the object as opposed to the previous version, empty if this is a ods:Create or ods:Tombstone activity",,,"[{'op': 'add', 'path': '/ods:attributes/dwc:collectionId', 'value': 'Herbarium Berolinense'}, {'op': 'copy', 'path': '/ods:attributes/dwc:identification/0/taxonIdentifications/0/dwc:infragenericEpithet', 'from': '/ods:attributes/dwc:identification/0/taxonIdentifications/0/dwc:specificEpithet'}]",http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,Activity,False,False,prov:Activity.changeValue,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/changeValue,ods:changeValue,array<object>,
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply app/data/opends/create-update-tombstone-event-termlist.csv:10]: Phhff no focussed to much on DigitalSpecimen and DigitalMedia. Going to check how to fix this

[the remark this answers is not in the document]
Comment (samleeflang) [reply app/data/opends/create-update-tombstone-event-termlist.csv:10]: Going to use `|` as separator

File: app/data/opends/digital-media-termlist.csv [hunk 10]
```diff
@@ -1,89 +1,89 @@
...
+dwc:,measurementAccuracy,Measurement Accuracy,https://rs.tdwg.org/dwc/terms/measurementAccuracy,,,"0.1, normal distribution with variation of 2 m",http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,Assertion,False,False,Assertion.measurementAccuracy,http://rs.tdwg.org/dwc/terms/,http://rs.tdwg.org/dwc/terms/measurementAccuracy,dwc:measurementAccuracy,string,
+dwc:,measurementUnit,Measurement Unit,https://rs.tdwg.org/dwc/terms/measurementUnit,,,"m, cm",http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,Assertion,False,False,Assertion.measurementUnit,http://rs.tdwg.org/dwc/terms/,http://rs.tdwg.org/dwc/terms/measurementUnit,dwc:measurementUnit,string,
+dwciri:,measurementUnit,Measurement Unit,https://rs.tdwg.org/dwc/iri/measurementUnit,,,http://vocab.nerc.ac.uk/collection/P06/current/UMSQ/,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,Assertion,False,False,Assertion.measurementUnit,http://rs.tdwg.org/dwc/iri/,http://rs.tdwg.org/dwc/iri/measurementUnit,dwciri:measurementUnit,string,
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply app/data/opends/digital-media-termlist.csv:48]: We will remove a most if not all hyperlinks for the descriptions (one of the comments in the openDS review). If we still got some left, we can look into making those a link.

File: app/data/opends/digital-media-termlist.csv [hunk 11]
```diff
@@ -1,89 +1,89 @@
...
-ods:,TombstonedByAgent,Tombstoned By Agent,"The agent who tombstoned the object, contains an ods:Agent object",,,,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,TombstoneMetadata,True,False,TombstoneMetadata.TombstonedByAgent,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/TombstonedByAgent,ods:TombstonedByAgent,object
-ods:,hasRelatedPID,Has Related Pid,The PIDs of the object the tombstoned object is related to,,,,http://www.w3.org/1999/02/22-rdf-syntax-ns#Property,TombstoneMetadata,False,False,TombstoneMetadata.hasRelatedPID,http://rs.dissco.eu/opends/terms/,http://rs.dissco.eu/opends/terms/hasRelatedPID,ods:hasRelatedPID,array
+namespace,term_local_name,label,definition,usage,notes,examples,rdf_type,class_name,is_required,is_repeatable,compound_name,namespace_iri,term_iri,term_ns_name,datatype,enum
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply app/data/opends/digital-media-termlist.csv:1]: I am not sure what you mean. The class_name is the Term Name and the full qualified name is with the ods prefix.
See https://dev.terms.dissco.tech/digital-specimen-terms#DigitalSpecimen the Term Name is Digital Specimen but the full qualified name is `ods:DigitalSpecimen`.

File: app/routes.py (hunk 1 above)
Comment (samleeflang) [reply app/routes.py]: Removed the logic for definition as this is never an array

File: app/data/opends/digital-media-termlist.csv (hunk 11 above)
[the remark this answers is not in the document]
Comment (southeo) [reply app/data/opends/digital-media-termlist.csv:1]: Ok then not sure which column this relates to, but on the web page, we see `belongs to class` which doesn't have the prefix. Should that have the prefix? 

https://dev.terms.dissco.tech/digital-specimen-terms#DigitalSpecimen_collectionID

**Review by southeo — APPROVED** [review]
:wheel: 
One outstanding comment, not blocking

File: app/templates/includes/termlist/termlist-table.html (hunk 2 above)
Comment (TomDijkema) [reply app/templates/includes/termlist/termlist-table.html:80]: I understand, just a generic HTML thing that is possible, but not recommended

File: app/data/opends/digital-media-termlist.csv (hunk 11 above)
[the remark this answers is not in the document]
Comment (samleeflang) [reply app/data/opends/digital-media-termlist.csv:1]: Yeah I see what you mean there. Not sure, LatimerCore als does it this way. Going to say it is fine for now, if we change it we might want to change it in the parent project as well

**Pull request merged**